### DISCUSSION TITLE
Confusion between Custom Group Node, Component, Template

### DISCUSSION THREAD

**Discussion by Mohsyn** [Comfy-Org/ComfyUI#3537] [Q&A] [answered]
It is slightly confusing that a selection of nodes can be saved as Custom Group Node, Component, and template? 
How are they different? where are they saved ?

**Comment by ltdrdata** [comment, marked as answer]
Template: A feature that allows you to save and load parts of a workflow.
(https://www.youtube.com/watch?v=ELDJxPTz3kw)

Group Node: A feature that allows you to convert parts of a workflow into a group for reuse within the same workflow.
(Due to its nature, it is belongs to the workflow. It cannot be used in other workflows.)

Component: This is not an official feature of ComfyUI. If referring to a component provided by ComfyUI-Manager, it is an application of the group node feature that allows group nodes to be saved to disk and reused across multiple workflows.
